### PR TITLE
Release v1.0

### LINKED ISSUE #63: Add release
The task is to tag final version of WPF client after whole work on it has been done and it was tested; it should be built in Release mode and binaries should be placed in the GitHub release.

### PR DESCRIPTION
**Pull request by gmrukwa**
Simple WPF client for DiviK algorithm.

Please do not squash this merge commit.

Resolves #63.

### CODE REVIEW

**Comment by gmrukwa** [comment]
[Add release](https://trello.com/c/wySBqHrk/81-add-release)

File: src/Spectre.Algorithms/Methods/Segmentation.cs [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   limitations under the License.
+*/
+using System;
+using Spectre.Algorithms.Parameterization;
+using Spectre.Algorithms.Results;
+using Spectre.Data.Datasets;
+
+namespace Spectre.Algorithms.Methods
+{
+	public class Segmentation: IDisposable
```
Comment (Sand3r-) [inline src/Spectre.Algorithms/Methods/Segmentation.cs]: This class misses documentation note

File: src/Spectre.Data/Datasets/IDataset.cs [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+using System.Collections.Generic;
+using Spectre.Data.Structures;
+
+namespace Spectre.Data.Datasets
+{
+    public interface IDataset
```
Comment (Sand3r-) [inline src/Spectre.Data/Datasets/IDataset.cs]: Missing brief note

File: src/Spectre.Data/Structures/DataPoint.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+namespace Spectre.Data.Structures
+{
+    public class DataPoint
```
Comment (Sand3r-) [inline src/Spectre.Data/Structures/DataPoint.cs]: Missing brief note

File: src/Spectre.Data/Structures/Metadata.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+namespace Spectre.Data.Structures
+{
+    public struct Metadata
```
Comment (Sand3r-) [inline src/Spectre.Data/Structures/Metadata.cs]: Missing brief note

File: src/Spectre.Data/Structures/SpatialCoordinates.cs [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+namespace Spectre.Data.Structures
+{
+    public struct SpatialCoordinates
```
Comment (Sand3r-) [inline src/Spectre.Data/Structures/SpatialCoordinates.cs]: Missing brief note

**Pull request merged**